mitiq/calibration/tests/test_calibration.py

```diff
@@ -378,11 +379,42 @@ def test_logging(capfd):
     )
     cal.run(log=True)
     captured = capfd.readouterr()
-    assert "ZNE results:" in captured.out
-    assert "PEC results:" in captured.out
-    assert settings.get_strategy(0).technique.name in captured.out
-    assert "noisy error" in captured.out
-    assert "mitigated error" in captured.out
+    for s in cal.strategies:
+        for line in str(s).split():
+            assert line in captured.out
+    for p in cal.problems:
+        for line in str(p).split():
+            assert line in captured.out
+    mcount = 0
+    ncount = 0
+    for line in captured.out.split("\n"):
+        if "Mitigated error: " in line:
+            mcount += 1
+        if "Noisy error: " in line:
+            ncount += 1
+    assert mcount == (len(cal.strategies) * len(cal.problems))
+    assert ncount == (len(cal.strategies) * len(cal.problems))
```

Comment:
This is checked in other parts of the tests. Feel free to drop it.

Reply:
This check makes sure that the logging output contains mitigated and noisy errors for all strategy/problem combination, i.e. one performance record per combination. I reworked this a little bit.